### PR TITLE
✨ k8s stateful's name is based on instance_name so we support more Re…

### PR DESCRIPTION
**Pull request by tomashejatko**
…dises in one project

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v1.2.0] - 2020-08-13
+### Changed
+- k8s stateful's name is based on instance_name so we support more Redises in one project
```
Comment (beranm14) [inline CHANGELOG.md]: Maybe:
```
Change K8s stateful set's name to contain `instance_name` postfix to support more Redis instances in one GCP project
```

**Comment by beranm14** [comment]
Otherwise LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (tomashejatko) [reply CHANGELOG.md]: You are only person using postfix and not suffix I've ever met :D Otherwise good idea, implemented

**Comment by beranm14** [comment]
Commit message also suffer from possibly unclear wording, sry, didn't noticed that

**Pull request merged**